src/askContainer/commands/cloneSkill/cloneHostedSkillManager.ts

```diff
@@ -41,29 +40,15 @@ export class CloneHostedSkillManager extends AbstractCloneSkillManager {
         throw new AskError("Failed to retrieve hosted skill repo URL from the service.");
       }
       await this.downloadGitCredentialScript(this.fsPath, this.context);
+   
       gitHelper.init();
       gitHelper.configureCredentialHelper(repoUrl, this.profile, skillId);
       gitHelper.addOrigin(repoUrl);
+      gitHelper.fetchAll();
+      gitHelper.checkoutBranch("prod")
+      gitHelper.checkoutBranch("master");
```

Comment:
huh?

Reply:
I see it was doing the same as well before.  I just don't understand why it's doing this branch change twice. some hack to pull both down locally I presume?

Reply:
Right, I believe the purpose is to pull both down locally and have the upstream set